pom.xml

```diff
@@ -27,11 +28,17 @@
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
         <java.level>6</java.level>
-        <workflow.version>1.4.3-beta-1</workflow.version>
+        <workflow.version>1.4.3</workflow.version>
     </properties>
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-step-api</artifactId>
+            <version>${workflow.version}</version>
```

Comment:
🐜  It feels wacky to use the workflow.version here when plugins are all independent versions.